resources/packs/core/port.go

```diff
@@ -175,6 +175,45 @@ func (p *mqlPorts) processesBySocket() (map[int64]Process, error) {
 	if c.Error != nil {
 		err = errors.New("cannot read related processess: " + c.Error.Error())
 	}
+
+	if len(res) == 0 {
```

Comment:
When does this fall back kicks in? Should we do an explicit platform check if it is for unix?

Reply:
Can we expect lsof on linux? I am only aware of its usage on BSD platforms.

Reply:
Ubuntu has it installed by default. Arch and Rocky have it in their repos. I can't remember a system where I had to install it. It's more complicated with `netstat` and `ss`. These vary between distros.

Reply:
We catch err on RunCommand. I assume the error shows that `lsof` isn't present or not in the `PATH`.

For the container case: That's more complicated. Matching pid and socket isn't that easy. Especially in the cases where we explicitly disable procfs and switch to commands. Sounds like an extra PR.

Reply:
I changed this to use `find`. This also works with containers:
```
cnquery run --sudo container 9df1d9ea2e4b -c "ports"                                                                                                                                                         
→ discover related assets for 1 asset(s)
→ resolved assets resolved-assets=1
! mql[processes]> could not retrieve process information error="process 4465 does not exist: /proc/4465" pid=4465
! mql[processes]> could not retrieve process information error="process 4471 does not exist: /proc/4471" pid=4471
ports.list: [
  0: port port=80 protocol="tcp" address="0.0.0.0" process.executable="nginx"
  1: port port=80 protocol="tcp" address="::" process.executable="nginx"
```
This is an nginx image running, which is based on debian.

Using `find` would also solve https://github.com/mondoohq/cnquery/issues/549 :
```
cnquery run --sudo ssh -i /home/christian/workspace/mondoo/examples/.vagrant/machines/default/virtualbox/private_key vagrant@127.0.0.1:2222 -c "ports" 
→ discover related assets for 1 asset(s)
→ resolved assets resolved-assets=1
ports.list: [
  0: port port=53 protocol="tcp" address="127.0.0.53" process.executable="/lib/systemd/systemd-resolved"
  1: port port=22 protocol="tcp" address="0.0.0.0" process.executable="sshd:"
  2: port port=22 protocol="tcp" address="10.0.2.15" process.executable="sshd:"
  3: port port=22 protocol="tcp" address="10.0.2.15" process.executable="sshd:"
  4: port port=22 protocol="tcp" address="10.0.2.15" process.executable="sshd:"
  5: port port=53 protocol="udp" address="127.0.0.53" process.executable="/lib/systemd/systemd-resolved"
  6: port port=68 protocol="udp" address="10.0.2.15" process.executable="/lib/systemd/systemd-networkd"
  7: port port=22 protocol="tcp" address="::" process.executable="sshd:"
  8: port port=80 protocol="tcp" address="::" process.executable="/usr/sbin/apache2"
]
```

Manually testing the `find` command on Alpine based images (BusyBox) also works. For cnquery, it would need some more testing after https://github.com/mondoohq/cnquery/issues/570 is fixed.